modin/engines/base/io/column_stores/feather_dispatcher.py

```diff
@@ -35,7 +35,7 @@ def _read(cls, path, columns=None, **kwargs):
         ----------
         path : str or file-like object
             The filepath of the feather file.
-        columns : array-like, optional
+        columns : array-like, default: None
```

Comment:
you can leave original version (`optional`) here, it is fine in this case I think

Reply:
Original version is left.